README.md

```diff
@@ -114,6 +114,39 @@ default maximum tile width. This can be modified in the `data` section of the tr
 it to a large file will let you zoom out further while still fetching data. This is useful for
 viewing low coverage BAM files.
 
+### Track options
+
+**colorScale** - Array that controls the color of substitutions and highlighted reads. It can take 6 or 11 values. 11 values are required if you want to control highlighted read colors (see the `highlightReadsBy` option). Example:
+```
+"colorScale": [
+  "#2c7bb6", //color of A substitutions
+  "#92c5de", //color of T substitutions
+  "#ffffbf", //color of G substitutions
+  "#fdae61", //color of C substitutions
+  "#808080", //color of N substitutions
+  "#DCDCDC", //color of other substitutions
+  "#FF0000", //color of reads with large insert size
+  "#0000D1", //color of reads with small insert size
+  "#00D1D1", //color of reads with LL orientation (see https://software.broadinstitute.org/software/igv/interpreting_pair_orientations)
+  "#555CFA", //color of reads with RR orientation 
+  "#02A221", //color of reads with RL orientation 
+]
+```
+
+**outlineReadOnHover** - Highlights the current read on hover.
+
+**outlineMateOnHover** - Highlights the mate of the current read on hover. If the mate is a split read, 
+both alignments will be highlighted.
+
+**highlightReadsBy** - Array that can take the values `insertSize`, `pairOrientation` or `insertSizeAndPairOrientation`:
+- if `insertSize` is set, reads that have a large or small insert size will be highlighted. The thresholds are controlled by the `largeInsertSizeThreshold` and `smallInsertSizeThreshold` track options. `largeInsertSizeThreshold` defaults to `1000`, i.e., 1000 bp. `smallInsertSizeThreshold` is not set by default, i.e, reads with small insert size won't be highlighted.
+- if `pairOrientation` is set, reads with an abnormal mapping orientation are highlighted (e.g. ++,--,-+).
+- if `insertSizeAndPairOrientation` is set, reads with an abnormal mapping orientation that also have abnormal insert sizes are highlighted.
+- if multiple values are set, reads that fulfill any of the conditions are highlighed in the corresponding color.
+- highlight colors can be controlled by extending the `colorScale` track option to 11 values. The additional 5 values will control the large insert size color, small insert size color and the ++, --, -+ mapping orientations (in that order).
```

Comment:
I have to say I'm not a huge fan of how we're implementing semantic colors in an array. This should be a dictionary that is indexed by strings to that it's legible.